src/app/components/PageLayoutWrapper/index.styles.ts

```diff
@@ -1,13 +1,13 @@
 import { css, Theme } from '@emotion/react';
 
 export default {
-  wrapper: ({ palette }: Theme) =>
+  wrapper: ({ isDarkUi, palette }: Theme) =>
     css({
       minHeight: '100vh',
       display: 'flex',
       flexDirection: 'column',
       justifyContent: 'space-between',
-      backgroundColor: palette.GHOST,
+      backgroundColor: isDarkUi ? palette.GREY_10 : palette.GHOST,
```

Comment:
Not sure if we should add the background colour at this level, as this component is added to all page types across the app. We would need to add the page type to the Theme array for the `darkUi` value to apply, but it could result in unexpected behaviour.

I think the reason there is a white bar across the top is due to the `margins` class applied in the `LiveTvPageLayout`. It appears to be pushing the content down from the top, causing the white bar to appear from behind. May be better to use a `padding` instead of `margin` for these? That will maintain the dark background from `pageWrapper`